src/irclog2html/irclog2html.py

```diff
@@ -952,7 +957,7 @@ def main(argv=sys.argv):
                      % (parser.prog, outfilename, e))
         try:
             parser = LogParser(infile, dircproxy=options.dircproxy)
-            formatter = style(outfile, colours)
+            formatter = style(outfile, outfilename, colours)
```

Comment:
This is the change that should fix tests:

```suggestion
            formatter = style(outfile, outfilename=outfilename, colours=colours)
```